app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -0,0 +1,76 @@
+<PowerSelect
+        @options={{this.departments}}
+        @searchEnabled={{false}}
+        @selected={{this.department}}
+        @triggerClass={{local-class 'select'}}
+        @onchange={{action this.onSelectChange}} as |department|
+>
+    {{department}}
+</PowerSelect>
+
+<ContentPlaceholders as |placeholder|>
+    <PaginatedList::HasMany
+            local-class='table'
+            @isTable=true
```

Comment:
```suggestion
            @isTable={{true}}
```
It "works" as you had it because it interprets `true` as a string, but is not actually correct. I'm *really* looking forward to typed-templates (we're working on it).

Reply:
Please fix.